### ISSUE TITLE
DEPRECATED: use your own version of lodash warning

### ISSUE THREAD

**Issue by dNitro** [amercier/karma-scss-preprocessor#61]
Using your plugin *karma* is throwing this warning message:
```bash
DEPRECATED: use your own version of lodash, this will go away in karma@2.0
```

**Comment by lousando** [comment]
I too have been experiencing this bug and I have traced the issue to [this](https://github.com/karma-runner/karma/blob/5c6b151d4a1f553b81e2b07202dbec09de95d680/lib/helper.js#L172):

When I do a stack trace, this is where the issue is arising in the plugin itself:
```
Trace: DEPRECATED: use your own version of lodash, this will go away in karma@2.0
    at Object.get (./myDir/node_modules/karma/lib/helper.js:173:13)
    at processFile (./myDir/node_modules/karma-scss-preprocessor/dist/lib/index.js:60:22)
    at nextPreprocessor (./myDir/node_modules/karma/lib/preprocessor.js:36:26)
    at ./myDir/node_modules/karma/lib/preprocessor.js:122:9
    at module.exports (./myDir/node_modules/isbinaryfile/index.js:28:12)
    at ./myDir/node_modules/karma/lib/preprocessor.js:84:7
    at ./myDir/node_modules/graceful-fs/graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:446:3)

```

So it's the use of the `helper` object

**Comment by lousando** [comment]
Made a PR to patch this issue #63

**Comment by darlanmendonca** [comment]
@amercier check the PR from @lousando pls

**Comment by darlanmendonca** [comment]
@lousando can you rename your branch? so anyeone can use your forked until @amercier see the PR.

e.g.

```
npm install --save-dev git+https://github.com/lousando/karma-scss-preprocessor#61
```

but how the branch names starts by number, npm dont work with that.
what u think? just while @amercier  dont look/accept your PR

**Comment by lousando** [comment]
@darlanmendonca I have gone with your suggestion and created this branch:

https://github.com/lousando/karma-scss-preprocessor/tree/lodash-fix

**Comment by darlanmendonca** [comment]
@lousando dont be better rename your current branch in PR `61` instead create a new? just for join all and future searchs to same branch of PR.

other think, the code is transpiled, when a dev install the module, npm load the files located in `dist`. @amercier do that in a prepublish script on npm, but if you want run transpile is

```sh
npm run transpile
```

but this generated files are in .gitignore, so we need to remove it from there to work when we install current fix from repository

while I wait for your reply, I replicate your changes, and remove dist from .gitignore, to use in my project

if you want my branch, and use it in a project, just

```sh
npm i --save-dev git+https://git@github.com/darlanmendonca/karma-scss-preprocessor.git#own-version-of-lodash
```

but if you want do this changes in your branch, and I believe that is better, do it pls, and I will use your branch too, instead replicate multiple times :D

**Comment by amercier** [comment]
Released in version 2.0.1, sorry for the long, long wait. Thanks @dNitro!
